shell/platform/darwin/common/framework/Headers/FlutterBinaryMessenger.h

```diff
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef FLUTTER_FLUTTERBINARYMESSENGER_H_
-#define FLUTTER_FLUTTERBINARYMESSENGER_H_
+#ifndef FLUTTER_SHELL_PLATFORM_DARWIN_COMMON_FRAMEWORK_HEADERS_FLUTTERBINARYMESSENGER_H_
+#define FLUTTER_SHELL_PLATFORM_DARWIN_COMMON_FRAMEWORK_HEADERS_FLUTTERBINARYMESSENGER_H_
```

Comment:
The public headers were intentional since users import them as `Flutter/FlutterBinaryMessenger.h` but in the end the symbol itself doesn't matter so long as it's unique (and ideally consistent), which this is, so lgtm.